src/modules/billing/services/charge-processor-service/lib/crm-helpers.js

```diff
@@ -76,13 +33,6 @@ const config = {
   }
```

Comment:
I'm not 100% sure. I assume so, because the file is barely doing anything else.

I was concerned that if I attempted to remove it, something ambiguous might break.

My IDE suggests that this is used only by:
`const [company, invoiceAccount, licenceHolderRole] = await crmHelpers.getCRMData(chargeVersion, chargePeriod.startDate);` 
in `water-abstraction-service/src/modules/billing/services/charge-processor-service/index.js`

Does that seem right?

Reply:
Yes that sounds right